README.md

```diff
@@ -31,7 +31,7 @@ Run the application with `rackup`.
 
 ### On Cloud.gov
 
-Compliance Viewer is deployed to cloud.gov.
+The 18F instance of Compliance Viewer is deployed to cloud.gov in the `cf` organization and `toolkit` space.
```

Comment:
cf is the cloud.gov organization.
On Mar 10, 2016 5:53 PM, "Jessie A. Young" <notifications@github.com> wrote:

> In README.md
> <https://github.com/18F/compliance-viewer/pull/6#discussion_r55763969>:
>
> > @@ -31,7 +31,7 @@ Run the application with `rackup`.
> >
> >  ### On Cloud.gov
> >
> > -Compliance Viewer is deployed to cloud.gov.
> > +The 18F instance of Compliance Viewer is deployed to cloud.gov in the `cf` organization and `toolkit` space.
>
> is there a cf org or is it 18f ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/compliance-viewer/pull/6/files#r55763969>.
>